packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -53,9 +90,40 @@ export function SquadPageHeader({
           <h1 className="text-center font-bold typo-title2 laptopL:text-left">
             {squad.name}
           </h1>
-          <h2 className="mt-1 text-center text-text-tertiary typo-body tablet:mt-2 laptopL:text-left">
-            @{squad.handle}
-          </h2>
+          <div className="mt-1 flex flex-row items-center justify-center text-text-quaternary tablet:mt-2 laptopL:justify-start">
+            <h2 className="text-center text-text-tertiary typo-footnote laptopL:text-left">
+              @{squad.handle}
+            </h2>
+            <Separator />
+            {createdAt && (
+              <span className="typo-caption2">Created {createdAt}</span>
+            )}
+          </div>
+          <div className="mt-4 flex flex-row items-center gap-2">
+            <Button
+              icon={props.icon}
+              size={ButtonSize.Small}
+              variant={
+                isFeatured ? ButtonVariant.Secondary : ButtonVariant.Float
+              }
+              className={
+                isFeatured &&
+                'relative border-overlay-primary-cabbage bg-overlay-tertiary-cabbage'
+              }
+            >
+              {props.copy} Squad
+              {isFeatured && (
+                <>
+                  <SparkleIcon className="absolute -top-2.5 right-0" />
```

Comment:
Not sure but was it intended for this to animate like sparkle randomly show and stuff? (Wont block this pr but maybe do find out)

Reply:
Yeah, let me check with the product team.

Reply:
It is holiday in IL right now, but once I get an answer, I will make an update if necessary.